src/pages/index.tsx

```diff
@@ -5,6 +5,13 @@ import Link from "next/link";
 export default function Home() {
     return (
         <Layout>
+            <meta charSet="UTF-8" />
```

Comment:
I'm not sure if using UTF-8 is even necessary. Probably not. Maybe though since you're displaying Japanese characters as well. Did you check if it works without it as well?

If it's actually necessary please use the correct casing of the attribute:
https://www.w3schools.com/TAGs/att_meta_charset.asp